.github/workflows/lint-validate.yml

```
@@ -29,18 +40,25 @@ jobs:
           python -m pip install --upgrade pip
           python -m pip install yamllint
 
+      # We use --no-warnings in this step to reduce output to critical errors
+      - name: Run yamllint on config/
+        run: |
+          yamllint -d scripts/yamllint-config.yaml --no-warnings config/
+
       - name: Unlock git-crypt secrets
+        if: env.fork == 'false'
         uses: sliteteam/github-action-git-crypt-unlock@a09ea5079c1b0e1887d4c8d7a4b20f00b5c2d06b
         env:
           GIT_CRYPT_KEY: ${{ secrets.GIT_CRYPT_KEY }}
 
       # We use --no-warnings in this step to reduce output to critical errors
-      - name: Run yamllint on config/ and secrets/config
+      - name: Run yamllint on secrets/config
+        if: env.fork == 'false'
         run: |
-          yamllint -d scripts/yamllint-config.yaml --no-warnings config/
           yamllint -d scripts/yamllint-config.yaml --no-warnings secrets/config/
 
   yamllint-templates:
+    needs: set-env-var
```

Something like this here
```suggestion
    needs: set-env-var
    if: github.event.pull_request.head.repo.fork
```

---

Ah! With `if: github.event.pull_request.head.repo.fork == 'false'` I don't need the set-env-var job at all!